eagleproject/core/migrations/0048_day_subscriber_alter_analyticsreport_created_at_and_more.py

```diff
@@ -0,0 +1,45 @@
+# Generated by Django 4.1.1 on 2023-02-21 14:44
+
+import datetime
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("core", "0047_rename_metastrat_holdings"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="Subscriber",
+            fields=[
+                (
+                    "id",
+                    models.AutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                ("email", models.EmailField(max_length=254, unique=True)),
+                ("conf_num", models.CharField(max_length=15)),
+                ("confirmed", models.BooleanField(default=False)),
+            ],
+        ),
+        migrations.AlterField(
+            model_name="analyticsreport",
+            name="created_at",
+            field=models.DateTimeField(
+                default=datetime.datetime(2023, 2, 21, 14, 44, 51, 970983)
```

Comment:
`created_at` & `updated_at ` already exist and have the `not null` property. And django ORM should be correctly setting these values without any need of doing it manually. Curious what is the reason for these default values?

Reply:
removed those bits